suite-native/module-dev-utils/src/components/FeatureFlags.tsx

```diff
@@ -17,6 +20,13 @@ export const FeatureFlags = () => {
                             onChange={toggleIsDeviceConnectFeatureFlagEnabled}
                         />
                     </Box>
+                    <Box flexDirection="row" justifyContent="space-between">
```

Comment:
Just an idea... Now when we have unified feature flags with an TypeScript enum, what about to render this whole component iteratively? Then whenever is a new FF added, we would not have to edit this file at all.

Something like:

```ts
 {featureFlags.map((featureFlag, index) => {
   <Box>
       <Text>{featureFlags[index]}</Text>
        <CheckBox
          isChecked={featureFlag}
          onChange={toggleFeatureFlag[featureFlag]}
        />
   </Box> 
})}
```

This code won't work it is just an proposal.

Reply:
Since this screen is visible only to developers, It would be fine for me to leave there just the feature flag name string as `isUsbDeviceConnectFeatureEnabled`. But understand if you don't like it.

Reply:
This is not only visible to developers. Production users can access this page as well, that's why I didn't do it. I thought about it as well, but creating a titles map would have to be done.

Reply:
you can replace enum by const object, that is more flexible

Reply:
According to this line are FFs displayed only in the Develop and Debug builds: https://github.com/trezor/trezor-suite/blob/078ad473cb4d461dbefd491934a69208aa0ee6eb/suite-native/module-dev-utils/src/screens/DevUtilsScreen.tsx#L24

Reply:
Oh yeah, my bad, I thought at least for android, it would be on production as well for testing purposes.

Reply:
https://github.com/trezor/trezor-suite/pull/10693/commits/42048c1fc0e4434c7a7e8acee95165039a561817